plugin/src/main/java/io/jenkins/plugins/coverage/model/visualization/colorization/CoverageLevel.java

```diff
@@ -0,0 +1,93 @@
+package io.jenkins.plugins.coverage.model.visualization.colorization;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+import io.jenkins.plugins.coverage.model.visualization.colorization.ColorProvider.DisplayColors;
+
+/**
+ * Provides the colorization for different coverage levels.
+ *
+ * @author Florian Orendi
+ */
+public enum CoverageLevel {
+
+    LVL_95(95.0, ColorId.OUTSTANDING),
+    LVL_90(90.0, ColorId.EXCELLENT),
+    LVL_85(85.0, ColorId.VERY_GOOD),
+    LVL_80(80.0, ColorId.GOOD),
+    LVL_75(75.0, ColorId.ABOVE_AVERAGE),
+    LVL_70(70.0, ColorId.AVERAGE),
+    LVL_65(65.0, ColorId.BELOW_AVERAGE),
+    LVL_60(60.0, ColorId.BAD),
+    LVL_50(50.0, ColorId.VERY_BAD),
+    LVL_0(0.0, ColorId.INSUFFICIENT),
+    NA(-1.0, ColorId.WHITE);
```

Comment:
Wouldn't it make sense to use a class here so that we make the percentage user configurable?

Reply:
Yes, I also think the colorization should be project specific. 

This enum - in long terms - is meant as a fallback colorization when no thresholds have been defined.
For further enhancements, I would suggest another color enum which is used to colorize coverage dependent on the set thresholds. 
This should act in a similar way, but without any further definitions but the thresholds required.

For example, if the general coverage threshold is 60% and the coverage is 65%, the color should be green for 'good' instead red for 'bad', like provided here.